feature-libs/quote/components/list/quote-list.component.html

```diff
@@ -1,111 +1,119 @@
-              </div>
-              <span [ngClass]="getQuoteStateClass(quote.state)">{{
-                'quote.states.' + quote.state | cxTranslate
-              }}</span>
-            </td>
-            <td role="cell" class="cx-updated-date">
-              <div class="cx-mobile-header">
+              </th>
+              <th role="columnheader" aria-sort="none">
                 {{ 'quote.list.updated' | cxTranslate }}
-              </div>
-              {{ quote?.updatedTime | cxDate: dateFormat }}
-            </td>
-            <td role="cell" class="cx-nav-caret">
-              <cx-icon [type]="iconTypes.CARET_RIGHT"></cx-icon>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-      <div class="cx-sort bottom">
-        <div
-          *ngIf="
-            quotesState.data.pagination.totalPages &&
-            quotesState.data.pagination.totalPages > 1
-          "
-        >
-          <cx-pagination
-            [pagination]="quotesState.data.pagination"
-            (viewPageEvent)="changePage($event)"
-          ></cx-pagination>
+              </th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody role="rowgroup">
+            <tr
+              role="row"
+              #row
+              *ngFor="let quote of quotesState.data.quotes"
+              [routerLink]="
+                {
+                  cxRoute: 'quoteDetails',
+                  params: { quoteId: quote.code }
+                } | cxUrl
+              "
+              (keydown.enter)="row.click()"
+              title="{{ quote.name }}"
+              [attr.aria-label]="'quote.list.clickableRow' | cxTranslate"
+            >
+              <td role="cell" class="cx-name">
+                <div class="cx-mobile-header">
+                  {{ 'quote.list.name' | cxTranslate }}
+                </div>
+                <span>{{ quote.name }}</span>
+              </td>
+              <td role="cell" class="cx-code">
+                <div class="cx-mobile-header">
+                  {{ 'quote.header.overview.id' | cxTranslate }}
+                </div>
+                {{ quote.code }}
+              </td>
+              <td role="cell" class="cx-status">
+                <div class="cx-mobile-header">
+                  {{ 'quote.header.overview.status' | cxTranslate }}
+                </div>
+                <span [ngClass]="getQuoteStateClass(quote.state)">{{
+                  'quote.states.' + quote.state | cxTranslate
+                }}</span>
+              </td>
+              <td role="cell" class="cx-updated-date">
+                <div class="cx-mobile-header">
+                  {{ 'quote.list.updated' | cxTranslate }}
+                </div>
+                {{ quote?.updatedTime | cxDate: dateFormat }}
+              </td>
+              <td role="cell" class="cx-nav-caret">
+                <cx-icon [type]="iconTypes.CARET_RIGHT"></cx-icon>
```

Comment:
Here the aria-label for the information about clicking the row should be added. Then we would get all the information of the row plus the hint for clicking.
<img width="630" alt="Screenshot 2023-11-02 at 11 13 54" src="https://github.com/SAP/spartacus/assets/54309425/4c68d4d3-cd8a-4a95-a770-f1a88f1cce52">